en/sensor/pmw3901.md

```diff
@@ -139,23 +141,22 @@ The AlienTek [ATK-PMW3901](https://www.aliexpress.com/i/32979605707.html) expose
 
 ![Alientek Pixhawk4 Connections](../../assets/hardware/sensors/pmw3901/ph4-alientek-flow-pinout.png)
 
-The board also incorporates a distance sensor (again, we recommend that you use an [external distance sensor](#external_rangefinder)).
-You can wire the internal sensor to the Pixhawk I2C port [in the same way as any other I2C peripheral](#i2c_wiring) 
-A screenshot showing the I2C pins (SLA, SLC, GND and VCC) is provided below.
+The board also incorporates a distance sensor (We recommend you use an [external distance sensor](#external_rangefinder)).
```

Comment:
We should be lower case as it is part of the sentence, also missing "that). Removal of the "again" is good. That kind of extra word is unnecessary.